### ISSUE TITLE
[Bug]: Custom events do not trigger

### ISSUE THREAD

**Issue by mkolodiy** [clientIO/joint#2382]
### What happened?

Custom events defined on elements via the `event` attribute do not trigger and therefore are not received in the callback defined via `paper.on` function. I followed the istructions defined on the https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#dia.attributes.event page.

In the following example I expect `link:delete` to be printed to the console when I click on the delete label but instead `link:pointerdown` is printed.

![Screenshot 2023-11-04 173742](https://github.com/clientIO/joint/assets/10558781/ee625e80-ae12-49ca-86f7-bb76426e19b1)

```
<!DOCTYPE html>
<html>
  <head>
    <link
      rel="stylesheet"
      type="text/css"
      href="https://cdnjs.cloudflare.com/ajax/libs/jointjs/3.7.5/joint.css"
    />
  </head>
  <body>
    <!-- content -->
    <div id="myholder"></div>

    <!-- dependencies -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.4/jquery.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.4.1/backbone.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jointjs/3.7.6/joint.js"></script>

    <!-- code -->
    <script type="text/javascript">
      var namespace = joint.shapes;

      var graph = new joint.dia.Graph({}, { cellNamespace: namespace });

      var paper = new joint.dia.Paper({
        el: document.getElementById('myholder'),
        model: graph,
        width: 600,
        height: 100,
        gridSize: 1,
        cellViewNamespace: namespace,
      });

      paper.on('link:pointerdown', (link, event) => {
        console.log('link:pointerdown');
      });

      paper.on('link:delete', (link, event) => {
        event.stopPropagation();
        console.log('link:delete');
      });

      const removeLabel = {
        markup: [
          {
            tagName: 'rect',
            selector: 'background',
          },
          {
            tagName: 'image',
            selector: 'buttonLabel',
          },
        ],
        attrs: {
          background: {
            fill: 'white',
            width: '24px',
            height: '24px',
            rx: 1,
            ry: 1,
            xAlignment: 52.5,
            yAlignment: -35,
          },
          buttonLabel: {
            event: 'link:delete',
            'xlink:href': `data:image/svg+xml;utf8,${encodeURIComponent(
              '<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0z" fill="none"/><path d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"/></svg>'
            )}`,
            xAlignment: 52.5,
            yAlignment: -35,
            title: 'Remove',
            cursor: 'pointer',
            width: 24,
            height: 24,
          },
        },
      };

      var rect = new joint.shapes.standard.Rectangle();
      rect.position(100, 30);
      rect.resize(100, 40);
      rect.attr({
        body: {
          fill: 'blue',
        },
        label: {
          text: 'Hello',
          fill: 'white',
        },
      });
      rect.addTo(graph);

      var rect2 = rect.clone();
      rect2.translate(300, 0);
      rect2.attr('label/text', 'World!');
      rect2.addTo(graph);

      var link = new joint.shapes.standard.Link();
      link.source(rect);
      link.target(rect2);
      link.addTo(graph);
      link.appendLabel(removeLabel);
    </script>
  </body>
</html>
```

### Version

3.7.6

### What browsers are you seeing the problem on?

Firefox, Chrome

### What operating system are you seeing the problem on?

Windows

**Comment by kumilingus** [comment]
This is just being addressed by this [PR](https://github.com/clientIO/joint/pull/2381).

**Comment by kumilingus** [comment]
Fixed in [v3.7.7](https://github.com/clientIO/joint/releases/tag/v3.7.7).
